src/services/deno_service.ts

```diff
@@ -79,12 +49,62 @@ export default class DenoService {
   }
 
   /**
-   * @description
-   * Get JSON file of 3rd party modules on deno.land
+   * Fetches the description for a module
+   *
+   *     await getThirdPartyDescription("drash"); // "A REST microframework ..."
+   *
+   * @param importedModuleName - The imported module in which we want to get the description for
    *
-   * @returns {DenoLandDatabase}
```

Comment:
No JSDOC

Reply:
I cant see this? I assume you were looking at another commit, but it what i see on `src/services/deno_service.ts` is:
![image](https://user-images.githubusercontent.com/47337480/89237511-459a6a80-d5eb-11ea-9ab4-b520256df03b.png)

Reply:
sorry was on my phone and didnt read the diff correctly